addons/binding/org.openhab.binding.heos/ESH-INF/thing/Channels.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="heos"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="control">
```

Comment:
Many of those channel types exist as system channel types, which should be used instead (see https://www.eclipse.org/smarthome/documentation/development/bindings/thing-definition.html#system-state-channel-types)

Reply:
I changed the channels "Control", "Artist" and "Titel by the system available channels. Haven't found other channels to replace with existing ones.